examples/widgets/check_buttons.py

```diff
@@ -32,7 +32,10 @@
 check = CheckButtons(
     ax=rax,
     labels=lines_by_label.keys(),
-    actives=[l.get_visible() for l in lines_by_label.values()]
+    actives=[l.get_visible() for l in lines_by_label.values()],
+    label_props={'color': ['k', 'r', 'g']},
+    frame_props={'edgecolor': ['k', 'r', 'g']},
+    check_props={'facecolor': ['k', 'r', 'g']},
```

Comment:
```suggestion
    label_props={'color': ['black', 'red', 'green']},
    frame_props={'edgecolor': ['black', 'red', 'green']},
    check_props={'facecolor': ['black', 'red', 'green']},
```

Reply:
I rewrote this to use the colours from the lines themselves.